brewtils/test/comparable.py

```diff
@@ -168,7 +169,7 @@ def _assert_wrapper(obj1, obj2, do_raise=False, **kwargs):
 
     """
     try:
-        _assert_equal(obj1, obj2, **kwargs)
+        _assert_equal(obj1, obj2, expected_type, **kwargs)
```

Comment:
Since `expected_type` is now keyword param it's probably better to not depend on positional ordering:

```python
_assert_equal(obj1, obj2, expected_type=expected_type, **kwargs)
```

Reply:
Good call.  Fixed.